.github/actions/setup-environment/action.yml

```diff
@@ -44,7 +44,11 @@ runs:
     - name: Configure
       run: |
         ./eden config add default
-        ./eden config set default --key=eve.accel --value=false
+        if [-n "$(lscpu | grep -oE "vmx|svm")" ]; then
```

Comment:
Can you please check line with e.g. https://www.shellcheck.net?;)

Reply:
This PR should be renamed as "My attempt to gain 3rd dan in bash" :D